README.md

```diff
@@ -0,0 +1,43 @@
+## Edit Discourse posts locally
+Discedit allows you to edit [Discourse](https://www.discourse.org/) posts in your favourite local text editor. It woks by pulling a post from Discourse, openning it in your favourite text editor and automatically pushing the edits. 
+
+## Setting up
+You will need to install [Go](https://golang.org/doc/install) and generate a personal global Discourse API key. If you don't have the rights to handle keys, talk to the administrator of your Discourse installation.
```

Comment:
Should probably separate that "global" note on a second sentence in this paragraph being more clear about how the API key should look like, as apparently this is confusing and people will get it wrong.